crates/graphql/src/schema/query_root.rs

```diff
@@ -338,32 +338,33 @@ impl QueryRoot {
     fn featured_listings(
         &self,
         context: &AppContext,
+        #[graphql(description = "Return listings only from these auction houses")]
+        auction_houses: Option<Vec<String>>,
+        #[graphql(description = "Return listings not from these sellers")]
+        seller_exclusions: Option<Vec<String>>,
```

Comment:
The type for PublicKey implements ToSQL so should be able to reference the arguments as public keys.

```suggestion
        auction_houses: Option<Vec<PublicKey<AuctionHouse>>>,
        #[graphql(description = "Return listings not from these sellers")]
        seller_exclusions: Option<Vec<PublicKey<Wallet>>>,
```